prac_06/guitars.py

```diff
@@ -0,0 +1,31 @@
+"""
+Program that uses the guitar class object
+Estimated completion time: 40-50 minutes
+Actual completion time: 55 minutes
+"""
+from prac_06.guitar import Guitar
+
+guitars = []
+
+print("My guitars! ")
+guitar_name = input("Guitar Name: ")
+while guitar_name != '':
+    year = int(input("Year: "))
+    cost = float(input("Cost: "))
```

Comment:
changing this to float(input("Cost: $") might be better as it is indicating to the user simply that a monetary value should be input